pages/mesosphere/dcos/services/data-science-engine/1.0.1/custom-images/index.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: layout.pug
+navigationTitle: Custom Images 
+excerpt: Using Custom Images in DC/OS Data Science Engine
+title: Custom Images
+menuWeight: 7
+enterprise: true
+model: /mesosphere/dcos/services/data-science-engine/data.yml
+render: mustache
+---
+
+This guide will show you how to build a custom Docker image for Spark Executor.
+
+1. To identify the currently used image, open a **Terminal** from the Notebook UI and run the following command:
+
+    ```bash
+    cat /opt/spark/conf/spark-defaults.conf | grep "spark.mesos.executor.docker.image"
+    # Output would be something like this:
+    # mesosphere/jupyter-service-worker:26a3231f513a686a2fcfb6f9ddb8acd45da467b261311b48a45b2a55bb0f2613
+    ```
+
+1. Create a Dockerfile. On your personal laptop or server, create a file with name `Dockerfile` and put the following content into it:
```

Comment:
let's clarify that `conda install -yq spacy` is provided as an example and we install `spacy` for the demo purposes

Reply:
Changed to:

"1. Create a Dockerfile. We will use `conda install -yq spacy` as an example, and will install `spacy` for demo purposes. On your personal laptop or server, create a file with name `Dockerfile` and put the following content into it:"